dbt_project.yml

```diff
@@ -15,4 +17,19 @@ clean-targets:
 
 models:
     mailchimp_source:
-        materialized: ephemeral
\ No newline at end of file
+        materialized: view
```

Comment:
Can we add custom schema here

Iterable for ref: https://github.com/fivetran/dbt_iterable_source/blob/d2aa2b5b00f557ae21923f0f135e0ed6ce8687a1/dbt_project.yml#L8